pandas/tests/io/test_sql.py

```diff
@@ -879,11 +893,11 @@ def load_test_data_and_sql(self):
         create_and_load_iris_view(self.conn)
 
     def test_read_sql_view(self):
-        iris_frame = sql.read_sql_query("SELECT * FROM iris_view", self.conn)
+        iris_frame = sql.read_sql_query(self.text("SELECT * FROM iris_view"), self.conn)
```

Comment:
Is this something that we actually need to fix in ``read_sql_query`` itself?

str is one of the accepted values for the ``sql`` parameter.

Reply:
Can you open an issue about this? I would prefer pushing this to the user, because otherwise we would have to deal with SQLite without alchemy vs cases with alchemy